content/blog/testing-kafka-connectors.asciidoc

```diff
@@ -0,0 +1,317 @@
+---
+title: "Testing Kafka Connectors"
+date: 2022-08-25T09:20:00+01:00
+draft: false
+markup: adoc
+---
+:source-highlighter: rouge
+:rouge-style: base16.dark
+:icons: font
+:imagesdir: /images
+ifdef::env-github[]
+:imagesdir: ../../static/images
+endif::[]
+
+https://kafka.apache.org/documentation/#connect[Kafka Connect] is a key factor for the wide-spread adoption of Apache Kafka:
+a framework and runtime environment for connectors,
+it makes the task of getting data either into Kafka or out of Kafka solely a matter of configuration,
+rather than a bespoke programming job.
+There's dozens, if not hundreds, of readymade source and sink connectors,
+allowing you to create no-code data pipelines between all kinds of databases, APIs, and other systems.
+
+There may be situations though where there is no existing connector matching your requirements,
+in which case you can implement your own custom connector using the Kafka Connect framework.
```

Comment:
Maybe link here to https://kafka.apache.org/documentation/#connect_development ?

Reply:
Good idea, will do.